index.js

```diff
@@ -65,7 +65,7 @@ Ora.prototype.render = function () {
 };
 
 Ora.prototype.start = function () {
-	if (!this.enabled) {
+	if (!this.enabled || this.id) {
```

Comment:
Those are not identical. The first time, enabled is `false` and id is `undefined` which will result in `false` and because of the `!` will result in `true` and the spinner won't start. Did you mean

```js
if (this.enabled && this.id) {
```

If the spinner is enabled AND we have an id, do nothing.